grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy

```diff
@@ -69,7 +69,7 @@ class Invoice implements Serializable {
         invoiceItems cascade: "all-delete-orphan"
     }
 
-    static transients = ['vendorInvoiceNumber', 'totalValue', 'totalValueNormalized', 'documents', 'status']
+    static transients = ['vendorInvoiceNumber', 'totalValue', 'totalValueNormalized', 'documents', 'status', 'hasPrepaymentAssociated']
```

Comment:
Change to `hasPrepaymentInvoice` to keep it consistent.

Reply:
But technically invoice does not have prepayment invoice (order does)